CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - when setting an IPv6 address using the `[DIR|SD|FD] Addresses` directive, now bareos only listens on IPv6 instead of both IPv4 and IPv6 on dual-stack. If you were using the `[DIR|SD|FD] Addresses` directive to create a dual-stack socket that would listen on both IPv6 AND IPv4, it will not work that way anymore. You should now also explicitly specify the IPv4 address in the directive [PR #882]
 
 ### Fixed
+- docs: Improve data-type Address, Addresses in CustomizingTheConfiguration [PR #968]
```

Comment:
maybe that was "changed" and not "fixed", but your decision.

Reply:
Once upon a time it was fixed (Addresses example) But it has mutated into changed so yes better to move that.